paypal/paypal/helper.php

```diff
@@ -53,6 +53,11 @@ public function Storelog($name, $logdata)
 		$options = "{DATE}\t{TIME}\t{USER}\t{DESC}";
 		$my      = JFactory::getUser();
 
+		if (empty($logdata['JT_CLIENT']))
```

Comment:
It client : who is using the plugin. Eg, com_quick2cart, com_jticketing etc
it is used to create log file name in Joomla's log folder.